gpu_bdb/queries/q08/gpu_bdb_query_08_dask_sql.py

```diff
@@ -0,0 +1,281 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, BlazingSQL, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+import os
+
+from bdb_tools.cluster_startup import attach_to_cluster
+import cupy as cp
+import numpy as np
+import cudf
+
+from bdb_tools.utils import (
+    benchmark,
+    gpubdb_argparser,
+    run_query,
+)
+
+from bdb_tools.readers import build_reader
+
+from dask.distributed import wait
+
+
+# -------- Q8 -----------
+q08_SECONDS_BEFORE_PURCHASE = 259200
+q08_STARTDATE = "2001-09-02"
+q08_ENDDATE = "2002-09-02"
+
+REVIEW_CAT_CODE = 6
+NA_FLAG = 0
+
+
+def get_session_id_from_session_boundary(session_change_df, last_session_len):
```

Comment:
Same, Wonder if we should add this in a common file that both`dask-cudf`  and `dask-sql` import